doc/classes/Viewport.xml

```diff
@@ -128,6 +128,12 @@
 				Returns the [Control] having the focus within this viewport. If no [Control] has the focus, returns null.
 			</description>
 		</method>
+		<method name="gui_get_mouse_over" qualifiers="const">
+			<return type="Control" />
+			<description>
+				Returns the [Control] which is the leaf node of mouse over behaviour within this viewport. If no [Control] has the cursor, returns null.
```

Comment:
What does this mean? I think this needs a more clear description as to the average user the meaning of "leaf node" isn't clear.

And in either way, the documentation and code uses US English 🙂 
```suggestion
				Returns the [Control] which is the leaf node of mouse over behavior within this viewport. If no [Control] has the cursor, returns null.
```